apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java

```diff
@@ -89,6 +89,12 @@ private boolean isHasRepeatKey(String[] newItems) {
       lineCounter++;
     }
 
+    keys.forEach(item -> {
```

Comment:
how about changing line 84 to `keys.add(kv[0].toLowerCase());`?

Reply:
In my opinion, this way would be more flexible. It won't limit the user to use all lowercase key in properties. :-P

Reply:
by changing the key to lowercase in line 84 won't affect the actual properties, it's only a way to check whether there are case-insensitive duplicated keys...

Reply:
Aha, I got this point. :-D